worlds/ladx/__init__.py

```diff
@@ -280,21 +282,35 @@ def create_items(self) -> None:
                     # Properly fill locations within dungeon
                     location.dungeon = r.dungeon_index
 
-        # For now, special case first item
-        FORCE_START_ITEM = True
-        if FORCE_START_ITEM:
-            self.force_start_item()
+        if self.options.tarins_gift == TarinsGift.option_local_progression:
+            self.force_start_item(itempool)
+
+        self.multiworld.itempool += itempool
 
-    def force_start_item(self):    
+    def force_start_item(self, itempool):

```

Comment:
Unsure how you want to handle this, but this ignores/overrides a user's `non_local_items`, allowing an item to be placed locally when it wasn't supposed to be local. It may be best to exclude anything in `non_local_items` from `possible_start_items`. Regardless, it is probably a good idea to print out a warning if the `for` loop finishes without any item being placed.